### DISCUSSION TITLE
The reason why the css breaks.

### DISCUSSION THREAD

**Discussion by heimish902** [mmistakes/minimal-mistakes#3412] [Q&A]
![image](https://user-images.githubusercontent.com/93975793/152127170-cc354fd7-13c5-4048-8312-12ede6755e8d.png)
It looks normal on the local server.

![image](https://user-images.githubusercontent.com/93975793/152127243-12c63ee5-b8ee-443d-8662-6b2e3833c31c.png)
But on the actual server, the css is broken and visible.

![image](https://user-images.githubusercontent.com/93975793/152127297-5f1aa67e-bdf9-4439-8342-0c702d3ac6c1.png)
My blog address is heimish902.github.io, so I set it as above.
Is there a problem with the setting?

![image](https://user-images.githubusercontent.com/93975793/152127718-a7ca4d10-2eb1-46e9-a578-03ec3bf93e02.png)
![image](https://user-images.githubusercontent.com/93975793/152127757-b4dca745-6ed0-4357-8a69-00e48578c27f.png)
It's the same as the folder address with the actual files, but I don't know why it can't be updated in the console window.

**Comment by iBug** [comment]
[The repository](https://github.com/heimish902/heimish902.github.io) is empty now. Does the problem still exist?

**Reply by heimish902** [reply]
I tried making it from the beginning, but it's still not working.

**Reply by iBug** [reply]
Try [the starter repo](https://github.com/mmistakes/mm-github-pages-starter) if you can't figure out what's going wrong. It's much easier this way.

**Reply by heimish902** [reply]
Thank you for teaching me another way.
But I'm curious what the problem is. :(
